src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -262,28 +262,23 @@ public void postCommitComment(@NonNull String hash, @NonNull String comment) thr
         }
     }
 
-    public void deletePullRequestApproval(String pullRequestId) throws IOException, InterruptedException {
```

Comment:
Removing it as its not used anywhere in the code. Also searched jenkinsci and its not used by any plugins.